cypress/e2e/signIn.cy.js

```diff
@@ -10,10 +10,10 @@ const homePage = new HomePageObject();
 describe('Sign In page', () => {
   let user;
 
-  before(() => {
+  beforeEach(() => {
     cy.task('db:clear');
```

Comment:
Clearing the database in the `beforeEach` hook might not be necessary for every test case. Consider whether this operation is needed for each test or if it can be moved to the `before` hook to optimize test performance.